table/strlimit.go

```diff
@@ -1,12 +1,13 @@
 package table
 
+import "github.com/mattn/go-runewidth"
+
 func limitStr(str string, maxLen int) string {
 	if maxLen == 0 {
 		return ""
 	}
-
-	if len([]rune(str)) > maxLen {
-		return string([]rune(str)[:maxLen-1]) + "…"
+	if runewidth.StringWidth(str) > maxLen {
+		return runewidth.Truncate(str, maxLen-1, "") + "…" // before str[:maxLen] + "…"
```

Comment:
```suggestion
		return runewidth.Truncate(str, maxLen-1, "") + "…"
```

Reply:
My bad, I forgot to delete it because I used it during development to take a look at the original truncate.